desktop/core/src/desktop/lib/raz/raz_client_test.py

```diff
@@ -163,19 +166,72 @@ def test_check_access_adls(self):
             'context': {}, 
             'operation': {
               'resource': {
-                'storageaccount': 'gethuestorageaccount', 
-                'container': 'demo-gethue-container', 
-                'relativepath': 'demo-dir1/customer.csv'
-              }, 
-              'resourceOwner': 'gethuestorageaccount', 
-              'action': 'read', 
-              'accessTypes': ['read']
+                'storageaccount': 'gethuestorage', 
+                'container': 'data', 
+                'relativepath': '/user/csso_hueuser'
+              },
+              'action': 'get-status', 
+              'accessTypes': ['get-status']
             }
           },
           verify=False
         )
         assert_equal(resp['token'], "nulltenantIdnullnullbnullALLOWEDnullnull1.05nSlN7t/QiPJ1OFlCruTEPLibFbAhEYYj5wbJuaeQqs=")
 
+
+  def test_handle_adls_req_mapping(self):
```

Comment:
Either we call it `test_handle_adls_action_types_mapping` and we add a list of actions there as we go, or we call it `test_handle_adls_list`.

First might allow us to keep it simpler.